static/templates/settings/linkifier-settings-admin.handlebars

```diff
@@ -41,13 +41,14 @@
         </p>
 
         <table class="table table-condensed table-striped admin_filters_table">
-            <tbody id="admin_filters_table">
+            <thead>
                 <th>{{t "Pattern" }}</th>
                 <th>{{t "URL format string" }}</th>
                 {{#if is_admin}}
                 <th class="actions">{{t "Actions" }}</th>
                 {{/if}}
-            </tbody>
+            </thead>
+            <tbody id="admin_filters_table" {{#if is_admin}}{{else}}class="required-text" data-empty="{{t 'No linkifiers set.' }}"{{/if}}></tbody>
```

Comment:
Should this say the same thing for admins when there aren't any  Not sure why you have the conditional here, logically.

(Also, @rishig did you have strings in mind for these messages?)

Reply:
Admins see the UI to add new links which non-admins do not see. I had added screenshots in my comment to show the difference between the two.

Reply:
These strings look good. We should check whether other tables include the period or not, and follow that convention.

Reply:
@rishig They do include periods. I checked that before adding the text.

Reply:
This is what it looks like for admins. Which one should I go with?

![Screenshot from 2019-06-03 00-14-00](https://user-images.githubusercontent.com/25329595/58765842-bcc21200-8594-11e9-944e-90ec4d6aabc9.png)

Reply:
Great! The way you had it originally is good. 

At some point we should standardize all of these settings pages, but that's probably a bigger project.

Reply:
Okay, so I changed it back to the way it was. Updated the screenshots in the opening post as well.